.readthedocs.yaml

```diff
@@ -3,7 +3,7 @@ version: 2
 build:
   os: ubuntu-22.04
   tools:
-    python: "3.10"
+    python: "3.11"
```

Comment:
grepping the docs dir I see some references to python 3.7; might be good to clean those up / tweak here.

Reply:
Will go and clean those up.